test/smoke.bats

```diff
@@ -27,12 +33,12 @@ assert_json_contains() {
   # discussion here: https://github.com/stedolan/jq/issues/24)
   local query="${1?} | select(. == null | not)"
 
-  [[ -v DEBUG_JSON ]] && jq <<< "${output}" >&3
+  [[ ! -z "${DEBUG_JSON}" ]] && jq <<< "${output}" >&3
```

Comment:
Ugh, sorry about this. I forgot that I installed a newer version of Bash using Homebrew....